public/manifest.json

```
@@ -0,0 +1,105 @@
+{
+  "name": "Pubky App",
+  "short_name": "Pubky",
```

The `id` field is recommended by Chrome for uniquely identifying your PWA across updates. Without it, changing `start_url` or `scope` can create duplicate installations.

```suggestion
  "short_name": "Pubky",
  "id": "/",
```